youtube_dl/extractor/nytimes.py

```diff
@@ -96,6 +98,43 @@ def get_file_size(file_size):
             'thumbnails': thumbnails,
         }
 
+    def _extract_podcast_from_json(self, json, page_id, webpage):
+        audio_data = self._parse_json(json, page_id, transform_source=js_to_json)['data']
+        
+        description = audio_data['track']['description']
+        if not description:
+            description = self._html_search_meta(['og:description', 'twitter:description'], webpage)
+
+        episode_title = audio_data['track']['title'].strip("‘’") # strip curlyquotes
```

Comment:
In retrospect I think you're correct, done. This was the first test case I had and I overgeneralized.